STREAMING.md

```diff
@@ -0,0 +1,19 @@
+#Streaming No Code
+Streaming no code is almost as easy as standard no code. You're still not doing any coding, but now you also
+adopt the policy of not going back to things you previously failed to code and adding code to them and trying
+not to dwell on other things you've written no code for in the past, all while being careful not to get in the 
+way of other people who are also trying not to do anything.
+
+Streaming allows you to write and deploy no code to systems in real time, where real time is defined as "fast enough
+that your boss never has to dog you about it being slow." Streaming no code is actually beyond real time -- it's done in
+no time, flat.
```

Comment:
So does this mean streaming no code is faster than lightspeed? Sounds like it implements quantum entanglement / "spooky action at a distance" between systems under the hood -- this is really revolutionary, bleeding-edge tech! I think the same underlying principle could make for a visionary geographically distributed system

Reply:
It is faster than light, but it is not faster than no-light, so we're good. Basically the laws of physics couldn't give two figs.